nearlib/signing/unencrypted_file_system_keystore.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+const keyDir = './neardev';
+const KeyPair = require('nearlib/signing/key_pair');
+const {promisify} = require('util');
+
+/**
+ * Unencrypted file system key store.
+ */
+class UnencryptedFileSystemKeyStore {
+    constructor(keyDir, networkId) {
+        this.keyDir = keyDir;
+        this.networkId = networkId;
+    }
+
+    /**
+     * Set a key for an account with a given id on a given network.
+     * @param {string} accountId 
+     * @param {string} keypair 
+     */
+    async setKey(accountId, keypair) {
+        if (!await promisify(fs.exists)(keyDir)){
```

Comment:
nit: I think it's better to promisify stuff in bulk on import, e.g. something like:

```js
const fs = ["exists", "mkdir"].reduce((acc, name) => {
    acc[name] = require('util').promisify(require('fs')[name]);
    return acc;
}, {}); 
```

Reply:
I'd prefer to punt on this for now due to volume of high priority stuff